sql/core/src/main/scala/org/apache/spark/sql/scripting/SqlScriptingExecutionNode.scala

```diff
@@ -110,6 +110,11 @@ trait NonLeafStatementExec extends CompoundStatementExec {
  * Conditional node in the execution tree. It is a conditional non-leaf node.
  */
 trait ConditionalStatementExec extends NonLeafStatementExec {
+  /**
+   * Used by CONTINUE HANDLER for all ConditionalStatementExec and by EXIT HANDLER for
```

Comment:
Let's make it more clean about what the flag interrupted mean. The current comment is a bit hard to understand.